backend/app/routers/datasets.py

```diff
@@ -226,6 +270,56 @@ async def get_dataset(dataset_id: str, db: MongoClient = Depends(dependencies.ge
     raise HTTPException(status_code=404, detail=f"Dataset {dataset_id} not found")
 
 
+@router.get("/{dataset_id}/summary.jsonld", response_model=DatasetOut)
```

Comment:
since this is under dataset, i can actually set break points and hit this route
<img width="958" alt="image" src="https://user-images.githubusercontent.com/13950475/196979629-90245a35-bbd6-424f-b3f3-267e05e90c3c.png">
<img width="585" alt="image" src="https://user-images.githubusercontent.com/13950475/196979660-84a0c1f7-f215-4f4c-92e0-1318b5cd86f3.png">


But then it errors out and you should investigate
<img width="569" alt="image" src="https://user-images.githubusercontent.com/13950475/196979717-957b4f20-f596-441b-a378-2f3c49fd7037.png">

Reply:
during a cleanup I make sure it is a DatasetOut to safe to call .json on it now